src/features/components/fields/DateInputField.js

```diff
@@ -0,0 +1,21 @@
+//rooms = [{id, name}, ...]
```

Comment:
Will we have more components that are not fields? Maybe we can move this to the components folder.

Reply:
I created a component that is not a field and I'm using it to display the room info in the /listRooms command. So we could have these fields into a field folder to separate both concepts.

Reply:
Good for me then :)